src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -0,0 +1,128 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.apache.commons.math3.util.FastMath;
+import java.io.File;
+import java.util.List;
+
+/**
+ * This tool takes in intervals via the standard arguments of {@link GATKTool} preprocesses them and creates bins that
```

Comment:
comma after` {@link GATKTool}` missing

Reply:
On second thought, let's decide whether "preprocessing" includes binning or not.  Your description here seems to indicate that binning occurs after preprocessing, whereas the name of the tool indicates that binning is part of preprocessing (otherwise, why not `PreprocessAndBinIntervals`?)

I'd rather just say that binning is a part of the preprocessing.  So for example, here I'd say something like `This tool takes in intervals via the standard arguments of {@link GATKTool}, prepares them for binning, and creates bins that cover the prepared intervals.`

Reply:
True, thanks guys.